runtime/cere-dev/src/lib.rs

```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48013,
+	spec_version: 48014,
```

Comment:
How about we don't increment spec_version on every PR but rather do that on releases?

Reply:
I propose to keep it as it is currently until we don't have a well-established and clear process on how we update Devnet runtime and runtime for the environments. I'm not sure what is the best way to deliver this fix to Devnet ASAP.